### PR TITLE
Anonymous user authentication

### LINKED ISSUE #15: Authentication
Users need to be able to login using both Kerberos and additional (plain Django) users.

### PR DESCRIPTION
**Pull request by invliD**
This PR adds a mapping from IP range to group ID that allows anonymous users to gain the permissions of the group if they are in the correct IP range.

Closes #15.

### CODE REVIEW

**Comment by Bartzi** [comment]
:ship:

File: _1327/settings.py [hunk 1]
```diff
@@ -73,12 +74,19 @@
...
 ]
 
 # needed by django-guardian library
 ANONYMOUS_USER_ID = -1
 GUARDIAN_RAISE_403 = True
 
+# Anonymous users in one of the given IP ranges are automatically assumed to be in the associated group.
+# Only the first match will be used.
+ANONYMOUS_IP_RANGE_GROUPS = {
+	# Example: '127.0.0.0/8': 2,
```
Comment (janno42) [inline _1327/settings.py]: could you do this by group name? we have defined `UNIVERSITY_GROUP_NAME` in the settings. and if we need a custom group to be added it's still easier to recognize by name.

**Comment by invliD** [comment]
@janno42: Done.

**Comment by janno42** [comment]
:+1:

**Pull request merged**